build-faucet.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run --rm -v $(pwd):/app golang:1.18 sh -c 'cd /app && GOOS=linux GOARCH=amd64 go build -o build/faucet .'
```

Comment:
Note that if this line fails the script still succeeds. The script should start with our prelude, but also because it's just one command being run:

```
#!/bin/bash
set -euo pipefail

exec docker run --rm -v $(pwd):/app golang:1.18 sh -c 'cd /app && GOOS=linux GOARCH=amd64 go build -o build/faucet .'
```